device/mellanox/x86_64-nvidia_sn5600_simx-r0/ACS-SN5600/sai_5600.xml

```diff
@@ -489,6 +489,13 @@
                 <breakout-modes>3</breakout-modes>
                 <port-speed>1536</port-speed>
             </port-info>
+            <port-info>
+                <local-port>257</local-port>
+                <width>1</width>
+                <module>64</module>
+                <breakout-modes>0</breakout-modes>
+                <port-speed>64</port-speed>
```

Comment:
What does 64 means?

Reply:
Decided on SAI side I will show you the translation function